### PR TITLE
Rename events

### LINKED ISSUE #3533: Improve all event names
Most event names are not QGVAR'ed and some use simple present instead of past tense or vice versa.
We should change that.

Backward compatibility could be achieved by subscribing to the corrected event and in a separate event handler the event could be forwarded using the old name and a message could be written to the RPT. Then in 3.6.0 remove the wrong name event and the separate event handler.

ref #3529

### PR DESCRIPTION
**Pull request by esteldunedain**
**When merged this pull request will:**
- Rename the following events
    - `ace_addedHeartRateAdjustment` -> `ace_heartRateAdjustmentAdded`
    - `ace_cargoAddedByClass` -> `ace_cargoAdded`
    - `ace_enteredCardiacArrest` -> `ace_cardiacArrestEntered`
    - `ace_itemAddedToTriageCard` -> `ace_triageCardItemAdded`
    - `ace_reload_linkedAmmo` -> ` ace_reload_ammoLinked`
    - `ace_reload_returnedAmmo` -> `ace_reload_ammoReturned`
    - `ace_treatmentSuccess` -> ` ace_treatmentSucceded`
    - `ace_addedCargoByClass` -> `ace_addCargo`
    - `ace_common_engineOn` -> `ace_common_setEngine`
    - `ace_explosives_clientRequestOrientations` -> `ace_explosives_sendOrientations`
    - `ace_explosives_serverSendOrientations` -> `ace_explosives_orientationsSent`
    - `ace_interaction_lampTurnOff` -> `ace_interaction_setLampOff`
    - `ace_interaction_lampTurnOn` -> `ace_interaction_setLampOn`
    - `ace_overheating_spareBarrelsLoadCoolest` -> `ace_overheating_loadCoolestSpareBarrel`
    - `ace_overheating_spareBarrelsSendTemperatureHint` -> `ace_overheating_sendSpareBarrelTemperatureHint`
- Close #3533

### CODE REVIEW

File: addons/cargo/functions/fnc_addCargoItem.sqf [hunk 1]
```diff
@@ -26,4 +26,4 @@ for "_i" from 1 to _amount do {
 };
 
 // Invoke listenable event
-["ace_cargoAddedByClass", [_itemClass, _vehicle, _amount]] call CBA_fnc_globalEvent;
+["ace_cargoByClassAdded", [_itemClass, _vehicle, _amount]] call CBA_fnc_globalEvent;
```
Comment (commy2) [inline addons/cargo/functions/fnc_addCargoItem.sqf]: `ace_cargoAdded` ???

File: addons/explosives/XEH_postInit.sqf [hunk 2]
```diff
@@ -29,16 +29,16 @@ if (isServer) then {
...
+        TRACE_1("requestOrientations received:",_logic);
         // Filter the array before sending it
         GVAR(explosivesOrientations) = GVAR(explosivesOrientations) select {
             _x params ["_explosive"];
             (!isNull _explosive && {alive _explosive})
         };
-        TRACE_1("serverSendsOrientations sent:",GVAR(explosivesOrientations));
-        [QGVAR(serverSendOrientations), [GVAR(explosivesOrientations)], _logic] call CBA_fnc_targetEvent;
+        TRACE_1("sendOrientations sent:",GVAR(explosivesOrientations));
+        [QGVAR(sendOrientations), [GVAR(explosivesOrientations)], _logic] call CBA_fnc_targetEvent;
```
Comment (commy2) [inline addons/explosives/XEH_postInit.sqf]: `orientationsSent` ?
past tense

File: addons/cargo/functions/fnc_addCargoItem.sqf (hunk 1 above)
Comment (esteldunedain) [inline addons/cargo/functions/fnc_addCargoItem.sqf]: Yes, that'd probably be better

File: addons/medical/functions/fnc_addToTriageCard.sqf [hunk 3]
```diff
@@ -39,4 +39,4 @@ if (!_inList) then {
     _log pushBack [_newItem, 1, CBA_missionTime];
 };
 _unit setVariable [QGVAR(triageCard), _log, true];
-["ace_itemAddedToTriageCard", [_unit, _newItem, _amount]] call CBA_fnc_localEvent;
+["ace_triageCardItemAdded", [_unit, _newItem, _amount]] call CBA_fnc_localEvent;
```
Comment (commy2) [inline addons/medical/functions/fnc_addToTriageCard.sqf]: `ace_triageCardAdded`
Item seems superfluous here.

File: addons/cargo/functions/fnc_addCargoItem.sqf (hunk 1 above)
Comment (jonpas) [inline addons/cargo/functions/fnc_addCargoItem.sqf]: Originally this even fired only when cargo was added by providing a classname, not sure if that's still the same though.

File: addons/medical/functions/fnc_addToTriageCard.sqf (hunk 3 above)
Comment (commy2) [inline addons/medical/functions/fnc_addToTriageCard.sqf]: Oh, nvm. I get it. It's something added to the triage card. derp.

File: addons/cargo/functions/fnc_addCargoItem.sqf (hunk 1 above)
Comment (esteldunedain) [inline addons/cargo/functions/fnc_addCargoItem.sqf]: @jonpas, as far as I understand the current code you can "add" cargo by classname, or "load" cargo by providing an object. I think the verbs are enough to differenciate both things.

Comment (jonpas) [inline addons/cargo/functions/fnc_addCargoItem.sqf]: Yup sounds good.

**Comment by commy2** [comment]
^ very good. That makes sense now.

**Comment by Glowbal** [comment]
#4001

**Pull request closed without merging**